presto-orc/src/main/java/com/facebook/presto/orc/reader/ListSelectiveStreamReader.java

```diff
@@ -572,7 +572,14 @@ public Block getBlock(int[] positions, int positionCount)
             nextPosition = positions[positionIndex];
         }
 
-        return ArrayBlock.fromElementBlock(positionCount, Optional.ofNullable(nullsCopy), offsetsCopy, elementStreamReader.getBlock(elementOutputPositions, elementOutputPositionCount));
+        Block elementBlock;
```

Comment:
Looks like an unrelated change

Reply:
There are test failures without this change

Reply:
What are those failures? If the failure is not related to the session property, maybe we could separate them out into other commits?